pom.xml

```diff
@@ -308,6 +309,14 @@
 				<artifactId>postgresql</artifactId>
 				<version>${postgresql.version}</version>
 			</dependency>
+
+			<!-- https://mvnrepository.com/artifact/org.apache.activemq/activemq-core 5.7.0 -->
+			<dependency>
+				<groupId>org.apache.activemq</groupId>
+				<artifactId>activemq-core</artifactId>
```

Comment:
activemq-core is pretty old, it's better to use activemq-client in latest version probably.